package.json

```diff
@@ -64,7 +64,8 @@
     "release:publish:dry-run": "pnpm publish --recursive --tag latest --registry=\"http://localhost:4873/\"",
     "release:tag": "node scripts/releaseTag.mjs",
     "validate": "concurrently \"pnpm prettier && pnpm eslint\" \"pnpm proptypes\" \"pnpm docs:typescript:formatted\" \"pnpm docs:api\"",
-    "clean:node_modules": "rimraf --glob \"**/node_modules\""
+    "clean:node_modules": "rimraf --glob \"**/node_modules\"",
+    "test:performance:charts": "pnpm --filter @mui-x-internal/performance-charts test:performance"
```

Comment:
I don't see it used any where in the PR
```suggestion
    "clean:node_modules": "rimraf --glob \"**/node_modules\""
```

Reply:
It is used by the gh action

Reply:
Ok, the Ctrl+F does not work well on github 😅

But why adding a new command line instead of doing directly `pnpm --filter @mui-x-internal/performance-charts test:performance`

Reply:
Easier to run the tests locally 😅, but can remove it

Reply:
Will depends about how test splitting goes, but I assume we will see more and more of those `--filter`